### PR TITLE
feat(cli): remove RJSON

### PR DESCRIPTION
**Pull request by ematipico**
## Summary

- removed `RJSON` from the `init` command and the documentation;
- added a `-y` flag to the `init` command (a la npm) that bootstrap files without asking questions to the user. By default is `false`;
- removed syntax support from the VS Code extension;

## Test Plan

CI

### CODE REVIEW

File: internal/core/client/commands/init.ts [hunk 1]
```diff
@@ -46,64 +55,61 @@ export default createLocalCommand({
...
-			identSizeAsNumber = 1;
+
+			identSizeAsNumber = Number(indentSize);
+			if (
+				isNaN(identSizeAsNumber) ||
+				identSizeAsNumber === 0 ||
+				identSizeAsNumber > 10
+			) {
+				client.reporter.warn(
+					markup`You inserted a value that is not a number o is greater than 10. Rome will fallback to the default value (1).`,
```
Comment (yassere) [inline internal/core/client/commands/init.ts:106]: Small typo: `number o is greater` -> `number or is greater`

File: website/src/_includes/docs/project-config.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 **Rome** needs to know how to find your project and what files it includes. To do this we require a project configuration file.
 
-Your configuration can be placed in a [few different locations](#supported-locations), but we recommend using a single `rome.rjson` file. This file is written using [RJSON](#rome-json) which is our flavor of JSON. It supports comments and has a simpler syntax.
+Your configuration can be placed in a [few different locations](#supported-locations), but we recommend using a single `rome.json` file. This file is written using [RJSON](#rome-json) which is our flavor of JSON. It supports comments and has a simpler syntax.
```
Comment (leo-terratrue) [inline website/src/_includes/docs/project-config.md:5]: looks like this sentence would still need to be removed: ```This file is written using [RJSON](#rome-json) which is our flavor of JSON. It supports comments and has a simpler syntax.```

Comment (leothorp) [reply website/src/_includes/docs/project-config.md:5]: Also purely for my own learning, I would be curious to know what the rationale was behind the decision. RJSON seemed like a cool idea, but was it just deemed too idiosyncratic/nonstandard for practical purposes? Or was there another reason? Thank you.

Comment (ematipico) [reply website/src/_includes/docs/project-config.md:5]: I quote here [Sebastian's response](https://discord.com/channels/678763474494423051/678763474930761739/840272157854138398) from Discord:

> non-standard and it's an additional thing to learn to use rome. if we don't make it the recommended way and promote it then there's not a lot of value in maintaining it.

**Pull request merged**